pypuppetdb/api/__init__.py

```diff
@@ -808,5 +819,8 @@ def reports(self, **kwargs):
                 status=report['status'],
                 noop=report['noop'],
                 metrics=report['metrics']['data'],
-                logs=report['logs']['data']
+                logs=report['logs']['data'],
+                code_id=code_id,
+                catalog_uuid=catalog_uuid,
+                cached_catalog_status=cached_catalog_status
```

Comment:
This might be better as the following.
```
code_id=report.get('code_id')
catalog_uuid=report.get('catalog_uuid')
cached_catalog_status=report.get('cached_catalog_status')
```